demos/golang/grpc_pingpong/run_pong_on_occlum.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+
+occlum_pong="./occlum_pong"
+
+if [ ! -f $occlum_pong ];then
+    echo "Error: cannot stat file '$occlum_pong'"
+    echo "Please see README and build it using prepare_pong_pong.sh"
+    exit 1
+fi
+
+# Init Occlum Workspace
+rm -rf occlum_pong_context && mkdir occlum_pong_context
```

Comment:
We have renamed `occlum context` to `occlum instance` in 0.15.0, use `occlum_pong_instance` as the name